# not sure the reason for changing link order. Disable it for now, but keep for a while
#
# comment added by Kevin Tian <kevin.tian@intel.com>, 2010-07-10

Index: dbus-glib-0.82/dbus/Makefile.am
===================================================================
--- dbus-glib-0.82.orig/dbus/Makefile.am	2009-07-16 15:54:44.000000000 +0100
+++ dbus-glib-0.82/dbus/Makefile.am	2009-12-11 09:27:20.000000000 +0000
@@ -105,7 +105,7 @@
 
 dbus_bash_completion_helper_SOURCES =                 	\
 	dbus-bash-completion-helper.c
-dbus_bash_completion_helper_LDADD=$(DBUS_LIBS) $(DBUS_GLIB_LIBS) $(builddir)/libdbus-gtool.la -lexpat $(builddir)/libdbus-glib-1.la
+dbus_bash_completion_helper_LDADD=$(DBUS_LIBS) $(DBUS_GLIB_LIBS) $(builddir)/libdbus-gtool.la $(builddir)/libdbus-glib-1.la -lexpat
 
 
 EXTRA_DIST=dbus-gmarshal.list make-dbus-glib-error-switch.sh make-dbus-glib-error-enum.sh dbus-bash-completion.sh.in
